apisix/cli/apisix.lua

```diff
@@ -21,11 +21,12 @@ local pkg_path_org = package.path
 local apisix_home = "/usr/local/apisix"
 local pkg_cpath = apisix_home .. "/deps/lib64/lua/5.1/?.so;"
                   .. apisix_home .. "/deps/lib/lua/5.1/?.so;"
-local pkg_path = apisix_home .. "/deps/share/lua/5.1/?.lua;"
+local pkg_path_deps = apisix_home .. "/deps/share/lua/5.1/?.lua;"
+local pkg_path_env = ";" .. apisix_home .. "/?.lua;"
```

Comment:
pkg_cpath_org already ends with `;`? The double ";;" has special meaning in Lua package path.

Reply:
according to my test, pkg_path_org doesn't have a `;` at the end:
```
~ $ apisix init
/usr/local/openresty/luajit/bin/luajit /usr/local/apisix/apisix/cli/apisix.lua init
pkg_path_org: ./?.lua;/usr/local/openresty/luajit/share/luajit-2.1.0-beta3/?.lua;/usr/local/share/lua/5.1/?.lua;/usr/local/share/lua/5.1/?/init.lua;/usr/local/openresty/luajit/share/lua/5.1/?.lua;/usr/local/openresty/lua
jit/share/lua/5.1/?/init.lua
pkg_path_deps: /usr/local/apisix/deps/share/lua/5.1/?.lua;
pkg_path_env: ;/usr/local/apisixaa/?.lua;
```

the test code is:
```
local pkg_cpath_org = package.cpath                     
local pkg_path_org = package.path
                                                 
local apisix_home = "/usr/local/apisix"
local pkg_cpath = apisix_home .. "/deps/lib64/lua/5.1/?.so;"          
                  .. apisix_home .. "/deps/lib/lua/5.1/?.so;"       
                                                                           
local pkg_path_deps = apisix_home .. "/deps/share/lua/5.1/?.lua;"     
local pkg_path_env = ";" .. apisix_home .. "aa/?.lua;"
  
-- modify the load path to load our dependencies
package.cpath = pkg_cpath .. pkg_cpath_org
package.path  = pkg_path_deps .. pkg_path_org .. pkg_path_env     
print("pkg_path_org: " .. pkg_path_org)
print("pkg_path_deps: " .. pkg_path_deps)        
print("pkg_path_env: " .. pkg_path_env)
```

Reply:
So it is an environmental problem. The `pkg_path_org` on my side ends with `;`. Maybe we can add a check (as a double `;;` might cause problem)?

Reply:
adding a check is nice, i've pushed a new commit.